tests/modules/nf-core/mitohifi/findmitoreference/test.yml

```diff
@@ -0,0 +1,14 @@
+## TODO nf-core: Please run the following command to build this file:
+#                nf-core modules create-test-yml mitohifi/findmitoreference
```

Comment:
This command fails with the following error message:

`INFO     Running 'mitohifi/findmitoreference' test with command:                                                                                                                 
         nextflow run ./tests/modules/nf-core/mitohifi/findmitoreference -entry test_mitohifi_findmitoreference -c ./tests/config/nextflow.config -c                             
         ./tests/modules/nf-core/mitohifi/findmitoreference/nextflow.config --outdir /var/folders/p0/6z00kpv16qbf_bt52y4zz2kc0000gp/T/tmp9zlc7am9 -work-dir                      
         /var/folders/p0/6z00kpv16qbf_bt52y4zz2kc0000gp/T/tmp2hul0uxf                                                                                                            
CRITICAL Error running test workflow (exit code 1)                                                                                                                               
         N E X T F L O W  ~  version 22.10.6                                                                                                                                     
         Launching `./tests/modules/nf-core/mitohifi/findmitoreference/main.nf` [prickly_curran] DSL2 - revision: f478a3f645                                                     
         Module compilation error                                                                                                                                                
         - file :                                                                                                                                                                
         /Users/verenakutschera/nobackup/misc/hackathons/202303_nf-core/modules/tests/modules/nf-core/mitohifi/findmitoreference/../../../../../modules/nf-core/mitohifi/findmito
         reference/main.nf                                                                                                                                                       
         - cause: Unexpected input: '{' @ line 1, column 36.                                                                                                                     
            process MITOHIFI_FINDMITOREFERENCE {                                                                                                                                 
                                               ^                                                                                                                                 
                                                                                                                                                                                 
         1 error`

Reply:
This has finally worked! :-)

Reply:
Awesome!